airbyte-integrations/connectors/source-stripe/source_stripe/source.py

```diff
@@ -49,8 +52,11 @@
 class SourceStripe(ConcurrentSourceAdapter):
 
     message_repository = InMemoryMessageRepository(entrypoint_logger.level)
+    _SLICE_BOUNDARY_FIELDS_BY_IMPLEMENTATION = {
```

Comment:
This would mean adding a public interface to the class for something that is temporary (hopefully one day everything will be concurrent). Do you have a strong opinion on this?

Reply:
no strong opinion. That's reasonable